app/views/pages/index.html.haml

```diff
@@ -1 +1,34 @@
-= render 'layouts/nav'
+= render :partial => 'layouts/nav'
+
+.container.mt-5
+
+  %button.btn.btn-primary.mb-3(type="button") Share your Journey
```

Comment:
Personally, I prefer the hash-like haml syntax: `%button{ type: "button }` because it looks more Ruby-ish.